kernel/src/path.rs

```diff
@@ -157,9 +177,25 @@ impl<Location: AsUrl> ParsedLogPath<Location> {
         }))
     }
 
+    pub(crate) fn should_list(&self) -> bool {
```

Comment:
could you reference this in the docs for `listed_log_files` so that it's clear what its semantics are?